### PR TITLE
use fork when running binary server

### PR DESCRIPTION
**Pull request by KananAli96**

### CODE REVIEW

File: eyes-universal-core/src/main/java/com/applitools/eyes/selenium/universal/server/UniversalSdkNativeLoader.java [hunk 1]
```diff
@@ -75,67 +69,84 @@ private static void startProcess() {
...
-        nativeProcess = createProcess(path.toString());
-        readPortOfProcess(nativeProcess);
+      String pathInJar = getBinaryPath(os, suffix);
+      InputStream inputStream = getFileFromResourceAsStream(pathInJar);
+
+      try {
+        copyBinaryFileToLocalPath(inputStream, path);
       } catch (Exception e) {
-        port = DEFAULT_SERVER_PORT;
+        // probably file in specified path is running, continue to start process
```
Comment (ikseek) [inline eyes-universal-core/src/main/java/com/applitools/eyes/selenium/universal/server/UniversalSdkNativeLoader.java:82]: It's worth ensuring that the failure is the one we expect: the failure to overwrite existing file. Skipping unexpected exceptions is the road to complicated debugging.

File: eyes-universal-core/src/main/java/com/applitools/eyes/selenium/universal/server/UniversalSdkNativeLoader.java [hunk 2]
```diff
@@ -75,67 +69,84 @@ private static void startProcess() {
...
-  private static void getFirstLineAsPort(BufferedReader reader) throws IOException {
+  private static void getFirstLineAsPort(BufferedReader reader)  throws Exception {
     String temp;
     String lastLine;
-    while ((temp = reader.readLine()) != null) {
-      lastLine = temp;
-      port = lastLine;
-      break;
+    try {
+      while ((temp = reader.readLine()) != null) {
```
Comment (ikseek) [inline eyes-universal-core/src/main/java/com/applitools/eyes/selenium/universal/server/UniversalSdkNativeLoader.java:136]: Does it really return null? Why?

Comment (ikseek) [reply eyes-universal-core/src/main/java/com/applitools/eyes/selenium/universal/server/UniversalSdkNativeLoader.java:136]: Ah I see when EOF is reached it returns null. But I don't get why do you have loop here if it is executed only once. You just need to read the first and only line of the output, if there is no line it's an error - you don't know the port. **After** reading the first line you might have a loop that will read and discard any additional output that server might produce (but it shouldn't) until EOF which means server terminated (or closed it's stdout, but it doesn't do that).

File: eyes-universal-core/src/main/java/com/applitools/eyes/selenium/universal/server/UniversalSdkNativeLoader.java (hunk 1 above)
Comment (KananAli96) [reply eyes-universal-core/src/main/java/com/applitools/eyes/selenium/universal/server/UniversalSdkNativeLoader.java:82]: actually, we log any exception there, but what should be done if failure isn't a copying file?

**Pull request merged**